src/Viz.js

```diff
@@ -148,6 +149,7 @@ export default class Viz extends BaseClass {
     this._shape = constant("Rect");
     this._shapes = [];
     this._shapeConfig = {
+      ariaLabel: this._drawLabel,
```

Comment:
looking at the chrome element inspector, it doesn't appear this is getting applied to the shapes...